server.go

```diff
@@ -0,0 +1,21 @@
+package main
+
+import "github.com/gin-gonic/gin"
+
+var get = make(map[string]string) // Map with user value
+
+func main() {
+	s := gin.Default() // Creates a Gin server
+	s.GET("/hello", func(c *gin.Context) { // Returns JSON
+		c.JSON(200, gin.H{
+			"message": "Hello world!", // Display simple "Hello world!"
+		})
+	})
+	s.GET("/greet/:name", func(c *gin.Context) { // Returns JSON with an argument from the user
+		n := c.Params.ByName("name") // Get user value from :name
```

Comment:
Rename `n` to something more meaningful, e.g. `name`

Reply:
Can you also make `"name"` a global const?
```suggestion
		n := c.Params.ByName(NameParam) // Get user value from :name
```